string my_string
float64 time_stamp
